source/slang/slang-parameter-binding.cpp

```diff
@@ -1065,81 +1075,91 @@ static void addExplicitParameterBindings_GLSL(
     // the index/offset/etc.
     //
 
-    TypeLayout::ResourceInfo* resInfo = nullptr;
-    TypeLayout::ResourceInfo* foundResInfo = nullptr;
-
-    LayoutSemanticInfo semanticInfo;
-    semanticInfo.index = 0;
-    semanticInfo.space = 0;
+    const int kMaxResCount = 2;
+    const int kSubpassResInfo = 1;
+    const int kResInfo = 0;
```

Comment:
Just FYI, a common trick for this is using enum, which can also be used as an array size because they are compile time constant.
```
enum
{
  kResInfo,
  kSubpassResInfo,
  kMaxResCount
};
```